templates/default/native_ndb_backup.sh.erb

```diff
@@ -132,17 +123,14 @@ do
     check_exit_code $? "rsync from $i failed, aborting. Exit code: $?" 3
 done
 
+# Backing up the schemas
+$MYSQL_ROOT/bin/mysqldump -u root -S $MYSQL_SOCKET --no-data --triggers --routines --events --all-databases > "$backup_target"/schema.sql
```

Comment:
Something that I'm not sure is whether you should backup all databases including some system db such as `information_schema` and `ndbinfo`. In ndb-agent we exclude those along with some others https://github.com/logicalclocks/ndb-agent/blob/main/internal/eventhandlers/mysqld_event_handler.go#L582 I am not sure if it cause any issue when restoring the backup

Reply:
Yeah ok I see  in the SQL file the schemas get dropped and re-created while mysql is running - the problem is that the mysqldump utility is a PITA to handle in bash and doesn't have the `--exclude-databases` option.